testground/benchmark/benchmark/utils.py

```diff
@@ -39,7 +39,8 @@ def wait_for_port(port, host="127.0.0.1", timeout=40.0):
     start_time = time.perf_counter()
     while True:
         try:
-            with socket.create_connection((host, port), timeout=timeout):
+            with socket.create_connection((host, port), timeout=timeout) as sock:
+                sock.close()
```

Comment:
```suggestion
            with socket.create_connection((host, port), timeout=timeout):
```

the context manager should have closed it already.